cns/restserver/restserver.go

```diff
@@ -349,3 +352,9 @@ func (service *HTTPRestService) MustGenerateCNIConflistOnce() {
 		}
 	})
 }
+
+func (service *HTTPRestService) AttachSWIFTv2Middleware(middleware cns.SWIFTv2Middleware) {
+	service.MultitenantMiddleware = middleware
+	// add multitenant ipconfig validator function
+	service.ipConfigsValidators = append(service.ipConfigsValidators, middleware.Validator())
```

Comment:
continuing my thought from above that we don't need this extra `Validator` method, and could have a `Validate(req) (...)` instead - here, that would simply look like:
```suggestion
	service.ipConfigsValidators = append(service.ipConfigsValidators, middleware.Validate)
```

Reply:
yeah you are right, honestly I don't remember why I even had that method 🙂